File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1064.lua
n=0
positivos=0

for i=1,6,1 do
    valor = io.read("*number")
    if(valor>0) then
        positivos=positivos+1
        n=n+valor
    end
end

print(string.format("%d valores positivos\n%.1f", positivos, n/positivos))

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1064.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);

        double valor, n=0;
        int positivos=0, i;
    
        for(i=1;i<=6;i++){
            valor = ler.nextDouble();
            if(valor>0){
                positivos++;
                n+=valor;
            }
        }
    
        System.out.printf("%d valores positivos\n%.1f\n", positivos, n/positivos);

    }
}